sdk/types/futures.ts

```diff
@@ -244,7 +244,7 @@ export type FuturesOrderTypeDisplay =
 	| 'Market'
 	| 'Liquidation'
 	| 'Delayed'
-	| 'Delayed Offchain';
```

Comment:
We shouldn't change this underlying type but just make sure we map the naming for user display